Tests/SystemTests/PublisherAuthenticationSystemTests.swift

```diff
@@ -0,0 +1,171 @@
+import XCTest
+import AblyAssetTrackingCore
+import AblyAssetTrackingPublisher
+import AblyAssetTrackingInternal
+import CoreLocation
+import Ably
+
+class PublisherAuthenticationSystemTests: XCTestCase {
```

Comment:
Is there a reason to have separate tests for Publisher and Subscriber authentication? It introduces code duplication as both those classes are very similar :wink: Couldn't you merge them and create tests that check the authentication methods both on publisher and subscriber at the same time? We did something like that on [Android](https://github.com/ably/ably-asset-tracking-android/blob/main/integration-testing-app/src/androidTest/java/com/ably/tracking/tests/AuthenticationTests.kt)